src/components/Dashboard/MapView.presentation.jsx

```diff
@@ -1,34 +1,69 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { GoogleMap } from 'react-google-maps';
+import { geolocated, geoPropTypes } from 'react-geolocated';
+import { GoogleMap, Marker } from 'react-google-maps';
 import MarkerWithInfo from './MarkerWithInfo.presentation';
 
 class MapView extends Component {
   render() {
     const { adventures } = this.props;
 
+    const currentLocationIcon = {
+      path: 'M-6,0a6,6 0 6,0 12,0a6,6 0 1,0 -12,0',
+      fillColor: '#4285f4',
+      fillOpacity: 1,
+      scale: 1,
+      strokeColor: '#4285f4',
+      strokeWeight: 11,
+      strokeOpacity: 0.2
+    };
+
     return (
-      <GoogleMap
-        defaultZoom={7}
-        defaultCenter={{ lat: 46.542009, lng: -66.234974 }} >
-        {adventures.map((adventure) => {
-          let location = adventure.location;
+      <div>
+        <div className="MapWrapper">
+          <GoogleMap
+            defaultZoom={7}
+            defaultCenter={{ lat: 46.542009, lng: -66.234974 }} >
+            {!this.props.isGeolocationAvailable
```

Comment:
If geolocation isn't available or enabled by the user, it would be better suited to give the user a snackbar warning instead. Also the ternary inside of a ternary looks pretty awful and it would make more sense to do the user warning outside of the MapWrapper div and then just do a single statement check to display the marker.

I'm thinking something like:
```
const { adventures, coords, isGeolocationAvailable, isGeolocationEnabled } = this.props;
...
let geolocation = (isGeolocationAvailable && isGeolocationEnabled);
...
// in the jsx
{(geolocation && coords) &&
  <Marker
    position={{
      lat: coords.latitude,
      lng: coords.longitude,
    }}
    icon={currentLocationIcon}
  />}
...
// after MapWrapper div
{!geolocation &&
<Snackbar
  open={true}
  message="Location services need to be enabled to see current location"
  autoHideDuration={4000} />}
...
```